/**:
  ros__parameters:
    # Used by both noarm squat and wiggle arm examples.
    command_rate: 50.0        # Rate in Hz at which to send commands
    seconds_per_motion: 2.0   # Number of seconds per each motion of the robot.

    # Used by the noarm squat example.
    # The following joint angles are in the order of FL_hip_x, FL_hip_y, FL_knee, FR..., RL..., RR...
    stand_joint_angles: [0.12, 0.72, -1.45, -0.12, 0.72, -1.45, 0.12, 0.72, -1.45, -0.12, 0.72, -1.45]
    squat_joint_angles: [0.15, 1.30, -2.25, -0.15, 1.30, -2.25, 0.15, 1.30, -2.25, -0.15, 1.30, -2.25]

    # Used by the wiggle arm example.
    joints_to_wiggle:    [16,   18  ]   # Indices of the joints to wiggle. This corresponds to WR0 and F1X
    # Full list of indices -> joint name:
    # 0         1         2        3         4         5        6         7         8        9         10        11
    # FL_hip_x, FL_hip_y, FL_knee, FR_hip_x, FR_hip_y, FR_knee, RL_hip_x, RL_hip_y, RL_knee, RR_hip_x, RR_hip_y, RR_knee
    # 12   13   14   15   16   17   18
    # SH0, SH1, EL0, EL1, WR0, WR1, F1X
    wiggle_up_offsets:   [ 1.0, -0.5]   # Offset from starting position of each joints_to_wiggle for the first wiggle (up)
    wiggle_down_offsets: [-1.0, -0.5]   # Offset from starting position of each joints_to_wiggle for the second wiggle (down)
